Dockerfile

```diff
@@ -30,16 +39,15 @@ RUN apt-get update -qq && \
 USER "$USER"
 WORKDIR /home/"$USER"
 
-# The source of tue-env is already checked out in the current dir, moving this to the docker
-COPY / ./.tue/
+# Dynamic additions to Dockerfile in case of a PR
+##TUE_BEGIN
+##TUE_END
 
 # Setup tue-env and install target ros
     # Remove interactive check from bashrc, otherwise bashrc refuses to execute
 RUN sed -e s/return//g -i ~/.bashrc && \
-    # Change the ownership of tue dir to USER
-    sudo chown -R "$USER:$USER" ~/.tue/ && \
-    # Run the standard installation script
-    ~/.tue/installer/bootstrap.bash && \
+    # Run the standard installation script if not in a PR
+    source <(wget -q -O - https://raw.githubusercontent.com/tue-robotics/tue-env/"$CI_BRANCH"/ci/docker-build-select.bash) && \
```

Comment:
This is more of a bash thing than Docker.
@MatthijsBurgh @Timple The only thing I am a bit doubtful about is the branch I select for build selection.